Further simplifications

cristian-recoseanu opened the pull request:
- synchronisation to the most recent framework models
- more links to models

cristian-recoseanu committed:
Further simplifications

- synchronisation to the most recent framework models
- more links to models

jonathan-r-thorpe commented on docs/Exploring the device tree.md:
```diff
@@ -67,8 +66,7 @@ Example response from calling the generic getter (1m1) on the root block to retr
             "owner": 1,
-            "description": "Receiver monitor worker",
-            "constraints": null
+            "description": "Receiver monitor worker"
```
Perhaps instead of the Receiver Monitor, this should be the Device Manager?

- The Receiver Monitor is no longer part of the core spec
- This response is incorrect according to MS-05-02 (there should be a Class Manager and a Device Manager at least)

cristian-recoseanu replied:
I will update the response to include both managers.
I still think receiver monitor is a better example here because you wouldn't use FindMembersByPath to fetch managers in practice as they are always in the Root block.
Also, even though ReceiverMonitor is not in the core framework, it's still a "standard" class.

cristian-recoseanu committed:
Update Exploring the device tree.md

To correct the get members response which should include both managers as per MS-05-02.

cristian-recoseanu commented:
Reviewed on the call on the 30th of May 2023.
